Simplified.xcodeproj/project.pbxproj

```diff
@@ -2305,6 +2305,7 @@
 				73FCA2A825005BA4001B0C5D /* Sources */,
 				73FCA35A25005BA4001B0C5D /* Frameworks */,
 				73FCA38925005BA4001B0C5D /* Resources */,
+				732940C7251E75EA0065E362 /* Fix software licenses file */,
```

Comment:
this is a new build phase for OE that fixes on the fly the Software licenses file to say "Open eBooks" instead of SimplyE.